js/rpg_objects/Game_System.js

```diff
@@ -169,29 +169,24 @@ Game_System.prototype.onBattleEscape = function() {
     this._escapeCount++;
 };
 
-Game_System.prototype.onFrameUpdate = function() {
-    this._frameCount++;
-};
-
 Game_System.prototype.onBeforeSave = function() {
     this._saveCount++;
     this._versionId = $dataSystem.versionId;
     this._framesOnSave = Graphics.frameCount;
+    this._sceneFramesOnSave = SceneManager.frameCount();
     this._bgmOnSave = AudioManager.saveBgm();
     this._bgsOnSave = AudioManager.saveBgs();
 };
 
 Game_System.prototype.onAfterLoad = function() {
-    if (!this._frameCount) {
-        this._frameCount = this._framesOnSave;
-    }
```

Comment:
Please, do not erase this for save data compatibility.

If this does not exist, `SceneManager.setFrameCount(undefined)` may occur. 😢

Reply:
I think that to raise the complexity of scripts for save data compatibility is not good. Though I respect your policy.